budgeteer-web-interface/src/test/java/org/wickedsource/budgeteer/web/pages/template/templateimport/ImportTemplatesPageTest.java

```diff
@@ -122,17 +139,26 @@ void AllCorrectInputForm(){
         assertThat(tester.getFeedbackMessages(null)).hasSize(1);
     }
 
-    private File getExampleTemplate(){
+    private File getExampleTemplate() {
         try {
             return new File(getClass().getResource("exampleTemplate1.xlsx").toURI());
-        }catch (URISyntaxException e){
+        } catch (URISyntaxException e) {
+            e.printStackTrace();
+            return null;
+        }
+    }
+
+    private File getInvalidExampleTemplate() {
+        try {
+            return new File(getClass().getResource("invalidTemplate.txt").toURI());
+        } catch (URISyntaxException e) {
             e.printStackTrace();
```

Comment:
Im Test kann die Exception einfach geworfen werden. Der Catch-Block kann weg.

Reply:
I removed the catch-block.